code/game/objects/structures/morgue.dm

```diff
@@ -222,6 +222,9 @@
 		return
 	if (!ismob(O) && !istype(O, /obj/structure/closet/body_bag))
 		return
+	if (!(ishuman(user) || ismartian(user) || ismonkey(user)) && !isrobot(user))
```

Comment:
iscarbon would catch all of these except robot, seems more maintainable; if you are carbon you have hands